### DISCUSSION TITLE
SDXL embedding training guide please

### DISCUSSION THREAD

**Discussion by sadsilly** [bmaltais/kohya_ss#1278] [General]
can someone make a guide on how to train embedding on SDXL. 

i asked everyone i know in ai but i cant figure out how  to get past wall of errors. :( :( :( :(

**Comment by 5nail000** [comment]
I see on Civitai there are some negatives in the format of textual inversions under SDXL1.0, so technically it is possible)))))
But in all topics, and on the git and somewhere else ... only questions from the series "does this work already?"

I couldn't get past the caching latents by myself...
I even tried to do everything at a resolution of 512, but still an error...

**Comment by 5nail000** [comment]
I just started it up!!!!!!!

maybe this args is necessarily:
`--no_half_vae --bucket_reso_steps 64 --gradient_checkpointing --enable_bucket`

my full command:
`source /workspace/kohya_ss/venv/bin/activate && accelerate launch  --num_cpu_threads_per_process 2 /workspace/kohya_ss/sdxl_train_textual_inversion.py --output_name tineuroava --pretrained_model_name_or_path /workspace/stable-diffusion-webui/models/Stable-diffusion/sd_xl_base_1.0.safetensors --output_dir /workspace/outputs/ --dataset_config /workspace/dataset_config.toml --logging_dir /workspace/logs/ --resolution "512, 512" --token_string tokentineuroava --init_word tineuroava --max_train_epochs 15 --learning_rate 1e-3 --save_every_n_epochs 1 --prior_loss_weight 1.0 --keep_tokens 0 --num_vectors_per_token 1 --train_batch_size 1 --mixed_precision no --cache_latents --use_8bit_adam --use_object_template --no_half_vae --bucket_reso_steps 64 --gradient_checkpointing --enable_bucket`

**Comment by andupotorac** [comment]
@5nail000 Were you able to do it?
